src/app/pages/home/home.component.scss

```diff
@@ -0,0 +1,86 @@
+.btn {
+    width: 8rem;
+    border-radius: 0;
+    margin-top: 1rem;
+}
+
+.card-text {
+  margin: 0 !important;
+  margin-bottom: 0rem !important;
+}
+
+img {
+    width: 10rem;
+    height: 10rem;
+    padding: 0.5rem;
+}
+
+.navbar {
+  padding-top: 2rem;
+  padding-left: 11%;
+  padding-right: 11%;
+}
+
+.h1, h1 {
+    font-size: 4rem;
+    color: whitesmoke;
+}
+
+.logo {
+    width: 3rem;
+    height: 3rem;
+}
+
+.page-header {
```

Comment:
Good work @JamesMudidi, I notice the background image takes really long to load, could you minify it and use a minified version of it for faster rendering., You can check if this website can help https://tinypng.com/